addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -7802,12 +7806,43 @@ registry.VersionControl = SnippetOptionWidget.extend({
             onSuccess: snippetVersions => {
                 const isUpToDate = snippetVersions && ['vjs', 'vcss', 'vxml'].every(key => this.$target[0].dataset[key] === snippetVersions[key]);
                 if (!isUpToDate) {
-                    this.$el.prepend(qweb.render('web_editor.outdated_block_message'));
+                    this.$el.prepend(qweb.render('web_editor.outdated_block_message', {name: this.$target[0].dataset.name}));
+                    const snippetsMenuHeight = document.querySelector("#snippets_menu").offsetHeight;
+                    this.$el[0].style.setProperty("--snippets-menu-height", `${snippetsMenuHeight}px`);
```

Comment:
With this configuration, the alert block will cover the whole right panel which is blocking not only the snippet options, but also the page options too... e.g from a blog post page:

![master-rev--sobo_a](https://github.com/odoo/odoo/assets/25032876/3c6a9c5d-7881-42a7-b1a0-273ac73f14db)

Reply:
We should ask PDE to confirm but I think this is what we want :thinking: I am not sure if he tested this case and if he agrees with it